bot/code_review_bot/workflow.py

```diff
@@ -266,12 +266,10 @@ def find_issues(self, revision, group_id):
         assert "tasks" in tasks
         tasks = {task["status"]["taskId"]: task for task in tasks["tasks"]}
         assert len(tasks) > 0
-        logger.info(
-            "Loaded Taskcluster group", id=settings.try_group_id, tasks=len(tasks)
-        )
+        logger.info("Loaded Taskcluster group", id=group_id, tasks=len(tasks))
 
         # Update the local revision with tasks
-        revision.setup_try(tasks)
+        revision.setup_try(group_id, tasks)
```

Comment:
A little below, we are already getting the decision task with `task = tasks.get(settings.try_task_id)`. Maybe make `setup_try` return the decision task, so we have no duplicated code (and if we decide to change the way to detect the decision task, we can do it just once).

Reply:
Or, get the decision task here and pass it to setup_try too.

Reply:
the task designed by `settings.try_task_id` is not the decision task, but the code-review one. It's used to list all the analyzers with potential issues

Reply:
Oh, right.